### PR TITLE
ignore certain contigs when calling inter-contig pairs

### PR DESCRIPTION
**Pull request by tedsharpe**

### CODE REVIEW

**Comment by tedsharpe** [comment]
Only cuts the runtime down by 5%, but it produces the same vcf.

File: scripts/sv/fermiLite/scanBam.sh [hunk 1]
```diff
@@ -22,13 +22,15 @@ REF_FASTA="${MASTER_NODE}$5"
 REF_INDEX_IMAGE="$6"
 INTERVAL_KILL_LIST=$(echo "${REF_FASTA}" | sed 's/.fasta$/.kill.intervals/')
 KMER_KILL_LIST=$(echo "${REF_FASTA}" | sed 's/.fasta$/.kill.kmers/')
+ALTS_KILL_LIST=$(echo "${REF_FASTA}" | sed 's/.fasta$/.kill.alts/')
```
Comment (cwhelan) [inline scripts/sv/fermiLite/scanBam.sh]: The file /seq/references/Homo_sapiens_assembly38/v0/Homo_sapiens_assembly38.fasta.64.alt is missing 170 alt contigs that are in your kill list, though.

Comment (tedsharpe) [inline scripts/sv/fermiLite/scanBam.sh]: Just for the sake of documentation, even though we've already discussed it:  The missing contigs in the alt file stored in the reference directory do not list contigs that do not share substantial homology with one of chromosomal contigs.  Those aren't a problem for the mapper, so they're omitted.
We'll use the list of all non-chromosomal contigs for our SV cross-contig kill list.

**Comment by tedsharpe** [comment]
@SHuang-Broad Do you want an opportunity to review, or shall I just merge?

File: src/main/java/org/broadinstitute/hellbender/tools/spark/sv/FindBreakpointEvidenceSpark.java [hunk 2]
```diff
@@ -235,10 +249,13 @@ protected void runTool( final JavaSparkContext ctx ) {
             final Locations locations,
             final PipelineOptions pipelineOptions )
     {
+        Set<Integer> crossContigsToIgnoreSet;
+        if ( crossContigsToIgnoreFile != null ) crossContigsToIgnoreSet = Collections.emptySet();
```
Comment (tedsharpe) [inline src/main/java/org/broadinstitute/hellbender/tools/spark/sv/FindBreakpointEvidenceSpark.java]: Oh, wow.  So this has never actually been tested.  The 5% variation I saw was just noise.
Thanks, Steve.  I'll make this fix and test before merging.  Thanks for catching this.

Comment (SHuang-Broad) [inline src/main/java/org/broadinstitute/hellbender/tools/spark/sv/FindBreakpointEvidenceSpark.java]: I was wondering about this possibility actually :smirk:

Comment (SHuang-Broad) [inline src/main/java/org/broadinstitute/hellbender/tools/spark/sv/FindBreakpointEvidenceSpark.java]: Just approved since you've noticed this.

**Comment by cwhelan** [comment]
Yikes, sorry I didn't notice that in my review.

**Comment by tedsharpe** [comment]
Decreases runtime from 55 mins to 47.  Produces about the same number of variant calls, though some of them are a bit different.
I've rebased, squashed, and will merge when Travis and local testing have completed unless someone objects.

**Comment by cwhelan** [comment]
Sounds good to me.

**Pull request merged**